tests/loops/test_loop_state_dict.py

```diff
@@ -53,28 +53,25 @@ def test_loops_state_dict_structure():
                 "current": {"ready": 0, "started": 0, "processed": 0, "completed": 0},
             },
             "epoch_loop.scheduler_progress": {
-                "total": {"ready": 0, "started": None, "processed": None, "completed": 0},
-                "current": {"ready": 0, "started": None, "processed": None, "completed": 0},
+                "total": {"ready": 0, "completed": 0},
+                "current": {"ready": 0, "completed": 0},
             },
             "epoch_loop.batch_loop.optimizer_loop.state_dict": {},
             "epoch_loop.batch_loop.state_dict": {},
             "epoch_loop.batch_loop.optimizer_loop.optim_progress": {
                 "optimizer": {
-                    "step": {
-                        "total": {"ready": 0, "started": None, "processed": None, "completed": 0},
-                        "current": {"ready": 0, "started": None, "processed": None, "completed": 0},
-                    },
+                    "step": {"total": {"ready": 0, "completed": 0}, "current": {"ready": 0, "completed": 0}},
                     "zero_grad": {
-                        "total": {"ready": 0, "started": 0, "processed": None, "completed": 0},
-                        "current": {"ready": 0, "started": 0, "processed": None, "completed": 0},
+                        "total": {"ready": 0, "started": 0, "completed": 0},
+                        "current": {"ready": 0, "started": 0, "completed": 0},
                     },
                 },
                 "optimizer_idx": 0,
             },
             "epoch_loop.val_loop.state_dict": {},
             "epoch_loop.val_loop.dataloader_progress": {
-                "total": {"ready": 0, "started": None, "processed": None, "completed": 0},
-                "current": {"ready": 0, "started": None, "processed": None, "completed": 0},
```

Comment:
nice that these disappear. 
will loading the "old checkpoints" be agnostic to this change? but right now we are only storing it for the FT case right? so it's fine I guess :)

Reply:
Correct!